route/kafkamdm.go

```diff
@@ -113,6 +113,15 @@ func NewKafkaMdm(key string, matcher matcher.Matcher, topic, codec, schemasFile,
 	}
 
 	if saslEnabled {
+		if saslMechanism == "SCRAM-SHA-256" {
+			config.Net.SASL.Mechanism = sarama.SASLTypeSCRAMSHA256
+			config.Net.SASL.SCRAMClientGeneratorFunc = func() sarama.SCRAMClient { return &XDGSCRAMClient{HashGeneratorFcn: SHA256} }
+		} else if saslMechanism == "SCRAM-SHA-512" {
+			config.Net.SASL.Mechanism = sarama.SASLTypeSCRAMSHA512
+			config.Net.SASL.SCRAMClientGeneratorFunc = func() sarama.SCRAMClient { return &XDGSCRAMClient{HashGeneratorFcn: SHA512} }
+		} else {
+			log.Fatalf("Failed to reconize saslMechanism: '%s'", saslMechanism)
```

Comment:
this would break current usage of using sasl without specifying a mechanism.

Reply:
Good catch, the following commit fix the issue